starters/remix/app/integration/forms/ContactForm/ContactForm.tsx

```diff
@@ -0,0 +1,112 @@
+import { getFormProps, useForm } from '@conform-to/react'
+import { getZodConstraint, parseWithZod } from '@conform-to/zod'
+import { CheckCircle2, CircleAlert } from 'lucide-react'
+import { Input, Textarea } from '~/components/form'
+import { Alert, AlertDescription, AlertTitle } from '~/components/ui/alert'
+import { Button } from '~/components/ui/button'
+import { Label } from '~/components/ui/label'
+import { contactFormSchema } from '~/integration/forms/ContactForm/schema'
+import { useFetcher } from '@remix-run/react'
+import { action } from '~/routes/contact_form'
+
+export const ContactForm = () => {
+  const fetcher = useFetcher<typeof action>()
+
+  const [form, fields] = useForm({
+    id: 'contact-form',
+    lastResult: fetcher.data?.reply,
+    constraint: getZodConstraint(contactFormSchema),
+    onValidate({ formData }) {
+      return parseWithZod(formData, { schema: contactFormSchema })
+    },
+    shouldValidate: 'onBlur',
+  })
+
+  return (
+    <div className="flex items-center justify-center">
+      <div className="container mx-auto max-w-xl">
+        {fetcher.data?.reply.status === 'success' ? (
+          <div className="space-y-4">
+            <Alert>
+              <CheckCircle2 className="stroke-green-500" />
+              <AlertTitle
+                className="text-lg"
+                dangerouslySetInnerHTML={{
+                  __html:
+                    fetcher.data?.data?.submitWebform?.confirmation
+                      ?.confirmation_title || '',
+                }}
+              ></AlertTitle>
+              <AlertDescription
+                dangerouslySetInnerHTML={{
+                  __html:
+                    fetcher.data?.data?.submitWebform?.confirmation
```

Comment:
Can we remove this conditional object access if we validate to have the message in the function?